code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -309,6 +309,13 @@
 	tastes = list("crab" = 1)
 	foodtype = MEAT
 
+/obj/item/reagent_containers/food/snacks/meat/slab/chicken
+	name = "chicken meat"
+	desc = "A slab of raw chicken. Remember to wash your hands!"
+	cooked_type = /obj/item/reagent_containers/food/snacks/meat/steak/chicken
+	slice_path = /obj/item/reagent_containers/food/snacks/meat/rawcutlet/chicken
+	tastes = list("chicken" = 1)
```

Comment:
```suggestion
	tastes = list("salmonella" = 1)
```

Reply:
Won't this taste potentially carry over to the cooked steak?

Reply:
Oh probably